### PR TITLE
workflows: Add “Assign to one project” workflow

### PR DESCRIPTION
**Pull request by pdubroy**
- Add "Assign to one project" workflow for assigning new issues to particular project boards.
- ⚠️ I'm not super familiar with Github workflows. Made this one by following the instructions [here](https://github.com/marketplace/actions/assign-to-one-project).

### CODE REVIEW

**Comment by pdubroy** [comment]
@mrnugget ptal, since it seems you are at least somewhat familiar with Github workflows. :-)

File: .github/workflows/assign-to-one-project.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+name: Assign to one project
```
Comment (mrnugget) [inline .github/workflows/assign-to-one-project.yml:1]:
```suggestion
name: frontend-platform-project-inbox
```

I think that will be better in the actual CI runs.

**Review by mrnugget — APPROVED** [review]
Looks good to me! But I have to admit that my familiarity with GitHub Actions can be summed up as "I merge, then see whether it worked, then try something else, merge again, repeat until it works"

**Review by umpox — APPROVED** [review]
Great idea!

**Pull request merged**